scripts/establish_conda_env.sh

```diff
@@ -391,6 +391,10 @@ do
     --no-clean)
       ECE_CLEAN=1
       ;;
+    --set-python)
```

Comment:
This is not documented in display_usage

Reply:
Note that the documentation should mention it is only used if set mamba is true.

Reply:
I have added documentation to the `display_usage()`